core/templates/services/debug-info-tracker.service.ts

```diff
@@ -23,17 +23,17 @@ import { Injectable } from '@angular/core';
   providedIn: 'root'
 })
 export class DebugInfoTrackerService {
-  _sequenceOfActions: Array<any> = [];
+  _sequenceOfActions: Array<Object> = [];
```

Comment:
Isn't object too generic here? Ditto below

Reply:
Hey @nishantwrp, could you please address this?

Reply:
Please address this.

Reply:
Ditto.